v03_pipeline/lib/reference_data/clinvar.py

```diff
@@ -110,12 +112,25 @@ def parsed_and_mapped_clnsigconf(ht: hl.Table):
     )
 
 
+def get_clinvar_ht(
+    clinvar_url: str,
+    reference_genome: ReferenceGenome,
+):
+    etag = requests.head(clinvar_url, timeout=10).headers.get('ETag').strip('"')
+    try:
+        logger.info(f'Try using cached clinvar ht with etag {etag}')
+        ht = hl.read_table(clinvar_dataset_path(reference_genome, etag))
+    except hl.utils.FatalError:
```

Comment:
I think it's probably cleaner to use [hailtop.fs.exists](https://hail.is/docs/0.2/fs_api.html#hailtop.fs.exists) here.  The `FatalError` stands out to m!